.gitignore

```diff
@@ -39,7 +39,7 @@
 __pycache__/
 
 # Generated kernelspec
-share/jupyter/kernels/xcpp/kernel.json
+share/jupyter/kernels/**/*.json
```

Comment:
Why is this even in the ignore list?

Reply:
@vgvassilev I actually don't know. I assummed it was there for a reason, and generalised it. When I am next at a computer I will remove it, and change the PR name accordingly.

Reply:
I think we used to generate some files in the source directory with cmake configure_file. That’s gone now.